src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -223,18 +223,26 @@ public void AllowReadStreamBuffering_SetTrueThenGet_ExpectTrue(Uri remoteServer)
             Assert.True(request.AllowReadStreamBuffering);
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public async Task ContentLength_Get_ExpectSameAsGetResponseStream(Uri remoteServer)
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task ContentLength_Get_ExpectSameAsGetResponseStream(bool useSsl)
         {
-            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
-            using (WebResponse response = await request.GetResponseAsync())
-            using (Stream myStream = response.GetResponseStream())
-            using (StreamReader sr = new StreamReader(myStream))
+            var options = new LoopbackServer.Options { UseSsl = useSsl };
+
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
             {
-                string strContent = sr.ReadToEnd();
-                long length = response.ContentLength;
-                Assert.Equal(strContent.Length, length);
-            }
+                HttpWebRequest request = WebRequest.CreateHttp(uri);
+                request.ServerCertificateValidationCallback = (a, b, c, d) => true;
```

Comment:
This is ok, but for future reference you can avoid the "a, b, c, d" by using the anonymous method instead of lambda syntax, e.g. `delegate { return true; }`.

Reply:
I mimicked what was already there: https://github.com/dotnet/corefx/blob/master/src/System.Net.Requests/tests/HttpWebRequestTest.cs#L886. Thanks for the suggestion, good to know.